hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -3699,9 +3682,11 @@ public void prepareMiniBatchOperations(MiniBatchOperationInProgress<Mutation> mi
         if (mutation instanceof Put) {
           HRegion.updateCellTimestamps(familyCellMaps[index].values(), byteTS);
           miniBatchOp.incrementNumOfPuts();
+          miniBatchOp.incrementBytesForPuts(mutation.heapSize());
```

Comment:
Do we traverse any synchronize blocks here or update any volatiles? In other words, wondering what the cost of these updates are.

We are also bulking up metrics w/ these added numbers... the metrics we send the master on each report (every second or so).

Reply:
We spend a lot of cpu counting currently. This adds to the counting load.

Reply:
@saintstack Thanks for review
At present, we only have aggregate information and cannot know how much network resources are used in each region.